code/modules/assembly/bomb.dm

```diff
@@ -26,15 +26,14 @@
 	if(istype(W, /obj/item/device/analyzer))
 		bombtank.attackby(W, user)
 		return
-	if(W.iswrench() && !status)	//This is basically bomb assembly code inverted. apparently it works.
+	if((W.iswrench() && !status) || istype(W, /obj/item/wirecutters/bomb))	//This is basically bomb assembly code inverted. apparently it works.
```

Comment:
Does this actually work, should it be: `(!status && (W.iswrench() || istype(W, ...)))`